sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/binaryfile/BinaryFileFormat.scala

```diff
@@ -93,6 +98,68 @@ class BinaryFileFormat extends FileFormat with DataSourceRegister {
 
     val pathGlobPattern = binaryFileSourceOptions.pathGlobFilter
 
+    val modificationTimeFilters = new mutable.ListBuffer[Long => Boolean]
+    val lengthFilters = new mutable.ListBuffer[Long => Boolean]
+
+    def parseFilter(filter: Filter): Unit = {
```

Comment:
I would call it `createFilterFunctions` and let it return `Seq[FileStatus => Boolean]`